lib/sprite.dart

```diff
@@ -114,19 +60,13 @@ class Sprite {
   /// If the asset is not yet loaded, it does nothing.
   void renderCentered(Canvas canvas, Position p,
       {Position size, Paint overridePaint}) {
-    if (!loaded()) {
-      return;
-    }
     size ??= this.size;
     renderRect(canvas,
         Rect.fromLTWH(p.x - size.x / 2, p.y - size.y / 2, size.x, size.y),
         overridePaint: overridePaint);
   }
 
   void renderRect(Canvas canvas, Rect dst, {Paint overridePaint}) {
-    if (!loaded()) {
```

Comment:
I think we can replace loaded with a `hidden` method/prop. that could be used for the same thing if the user wants but is much more widely useful. a lot of our components have a custom if on render to hide things